internal/command/deploy/machines.go

```diff
@@ -481,6 +482,14 @@ func (md *machineDeployment) createOrUpdateReleaseCmdMachine(ctx context.Context
 	}
 }
 
+func defaultReleaseMachineGuest() *api.MachineGuest {
```

Comment:
now that we have clone, this can be fed from a top level variable + init() and we avoid the panic call